feather/common/src/entities/player.rs

```diff
@@ -17,7 +17,7 @@ pub fn build_default(builder: &mut EntityBuilder) {
 }
 
 /// The hotbar slot a player's cursor is currently on
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
```

Comment:
There's discussion for removing the default derive for this structure [here](https://github.com/feather-rs/feather/pull/471#issuecomment-927202657). Please leave your feedback and move forward with whatever we decide.

Reply:
We just sidestepped the whole problem, by removing the default implementation. It was only used in one place. We leave the decision to someone else to make.